pimd/pim6_mld.h

```diff
@@ -36,6 +36,14 @@ enum gm_sg_state {
 	GM_SG_NOPRUNE_EXPIRING,
 };
 
+#define gm_update_sg_state(sg)                                                 \
```

Comment:
Please use capital letters to define this macro. It is generally followed to differentiate the macros and functions easily. Also please add comments why is this change required.